content/docs/introduction/comparison.md

```diff
@@ -0,0 +1,66 @@
+|Inclusive and Extensible Ecosystem at Client or Server Levels      | <i class="fas fa-times-circle no"></i>       | <i class="fas fa-times-circle no"></i>        |  <i class="fas fa-check-circle yes"></i> |
+|Supports Multiple Management Servers                               | <i class="fas fa-times-circle no"></i>       | <i class="fas fa-check-circle yes"></i>       |  <i class="fas fa-check-circle yes"></i> |
+|Has Efficient Data Encoding                                        | <i class="fas fa-times-circle no"></i>       | <i class="fas fa-check-circle yes"></i>       |  <i class="fas fa-check-circle yes"></i> |
+|Has an Always-On Communication Mechanism                           | <i class="fas fa-times-circle no"></i>       | <i class="fas fa-check-circle yes"></i>       |  <i class="fas fa-check-circle yes"></i> |
+|Utilizes a Device:2 (TR-181i2) Driven Data Model                   | <i class="fas fa-times-circle no"></i>       | <i class="fas fa-check-circle yes"></i>       |  <i class="fas fa-check-circle yes"></i> |
+|Schema Drive Protocol Definition                                   | <i class="fas fa-check-circle yes"></i>      | <i class="fas fa-check-circle yes"></i>       |  <i class="fas fa-check-circle yes"></i> |
+|Defines a Robust Set of Operations                                 | <i class="fas fa-check-circle yes"></i>      | <i class="fas fa-check-circle yes"></i>       |  <i class="fas fa-check-circle yes"></i> |
+|May Leverage a Transport Layer Security Mechanism                  | <i class="fas fa-check-circle yes"></i>      | <i class="fas fa-check-circle yes"></i>       |  <i class="fas fa-check-circle yes"></i> |
+|Provides an Access Control Mechanism<sup>[1]</sup>                 | 1/2                                          | <i class="fas fa-check-circle yes"></i>       |  <i class="fas fa-check-circle yes"></i> |
+|Provides a Secure Clean/Screen Solution for 3rd Party Services     | <i class="fas fa-times-circle no"></i>       | <i class="fas fa-times-circle no"></i>        |  <i class="fas fa-check-circle yes"></i> |
+|Easy to Scale (100 - +50M CPEs)<sup>[3]</sup>                      | <i class="fas fa-times-circle no"></i>       | <i class="fas fa-question-circle maybe"></i>  |  <i class="fas fa-check-circle yes"></i> |
+|Easy to Scale (100 - +100K events/second)<sup>[3]</sup>            | <i class="fas fa-times-circle no"></i>       | <i class="fas fa-question-circle maybe"></i>  |  <i class="fas fa-check-circle yes"></i> |
+|Upgradable Without Outages at Scale<sup>[3]</sup>                  | <i class="fas fa-question-circle maybe"></i> | <i class="fas fa-question-circle maybe"></i>  |  <i class="fas fa-check-circle yes"></i> |
+|Functions Properly With Complete Datacenter Failure<sup>[3]</sup>  | <i class="fas fa-question-circle maybe"></i> | <i class="fas fa-question-circle maybe"></i>  |  <i class="fas fa-check-circle yes"></i> |
```

Comment:
It might be nice to explain how XMiDT accomplishes some of these things? Not sure that we want to include that on this page, but maybe another page where we go into some amount of detail explaining how we check each of these boxes and more information about how to set XMiDT up to accomplish these things.  It can help add credibility to our claims.

Reply:
Good points.  I'll see if I can do better with this.